packages/domain-picker/test/suggestion-item.tsx

```diff
@@ -15,293 +16,76 @@ import { render, fireEvent, screen } from '@testing-library/react';
 import '../matchMedia.mock';
 import SuggestionItem from '../src/domain-picker/suggestion-item';
 
-const testSuggestion = {
-	domain_name: 'example.com',
-	relevance: 0.9,
-	supports_privacy: true,
-	vendor: 'vendor',
-	cost: '€15.00',
-	product_id: 1234,
-	product_slug: '1234',
+const MOCK_PROPS = {
+	railcarId: 'id',
+	domain: 'example.com',
+	onSelect: jest.fn(),
+	onRender: jest.fn(),
 };
 
-/**
- * disabled the below test suite as these tests have been unmaintained whilst the codebase has moved on
- * and have now become stale. A separate task will be raised to fixed them. See https://github.com/Automattic/wp-calypso/issues/45501
- */
+const renderComponent = ( props = {} ): RenderResult =>
+	render( <SuggestionItem { ...MOCK_PROPS } { ...props } /> );
 
-beforeAll( () => {
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	// ( window as any ).configData = require( '../../../../../../config/test.json' );
-} );
+describe( 'traintracks events', () => {
+	afterEach( () => {
```

Comment:
Any reasons why you chose `afterEach` instead of `beforeEach`?

Reply:
What we want to achieve is to clear the mocks between assertions and for that it doesn't matter if it happens before or after. It just made more sense to me to clean up the mocks after :)

Reply:
I thought so, but I thought I'd ask anyway. Thank you!